### DISCUSSION TITLE
MPTABLE LAI source

### DISCUSSION THREAD

**Discussion by swesterma** [NCAR/hrldas#210] [Q&A]
Hi, I was wondering where the monthly LAI and SAI values in the look-up table MPTABLE are coming from? I could not find anything in the publications, technical descriptions or code comments. Is there any source for these values? Are they already based on satellite products such as MODIS? How were they disaggregated or merged to fit the distinction of PFTs?
I am thankful for any advice.
Cheers, Sven

**Comment by dmocko** [comment]
Hi @swesterma 

Thanks for asking this question.  I would also like an answer to this.
I did a search as well, and also wasn't able to find any information.

If possible, it would be great if sources were identified for all TABLE look-up parameters in the model.

**Comment by RAbolafiaRosenzweig** [comment]
From my understanding, this is from the MODIS 2000-2008 multiyear monthly average calculated by Yang et al. (2011) (https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2010JD015140). It would be good to update this moving forward.

**Reply by swesterma** [reply]
Thank you for the reply. As far as I understood, MODIS LAI in Yang et al. (2011) was only used for monthly evaluation of the dynamically predicted LAI but not for creating a climatology for different PFTs.

**Comment by cenlinhe** [comment]
Due to historical reason, there was unfortunately no good track of the sources for each look-up table parameter. To improve this, we have started to keep track of all new updates to the table parameters since the release of Noah-MP v5.0. Hope that in the future, we could further improve this.
